src/js/views/patients/sidebar/patient/patient-sidebar.scss

```diff
@@ -0,0 +1,14 @@
+.patient-sidebar__patient-info {
+  border-bottom: 1px solid $black-90;
+  padding: 16px 0;
+
+  .name {
```

Comment:
I think we should go ahead and un-nest these and give them BEM naming.  The styles can be whatever for now, but I don't want to add simple class names.

Reply:
Okay, I switched these to BEM naming. Will be included in the next commit.